vividus-to-xray-exporter/src/main/java/org/vividus/xray/converter/CucumberScenarioConverter.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.xray.converter;
+
+import static java.lang.System.lineSeparator;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.vividus.bdd.model.jbehave.Parameters;
+import org.vividus.bdd.model.jbehave.Scenario;
+import org.vividus.bdd.model.jbehave.Step;
+
+public final class CucumberScenarioConverter
+{
+    private static final String COMMENT_KEYWORD = "!-- ";
+    private static final String CUCUMBER_SEPARATOR = "|";
+
+    private CucumberScenarioConverter()
+    {
+    }
+
+    public static CucumberScenario convert(Scenario scenario)
+    {
+        if (scenario.getExamples() != null)
+        {
+            return new CucumberScenario("Scenario Outline", buildExampleScenario(scenario));
+        }
+        return new CucumberScenario("Scenario", getSteps(scenario));
```

Comment:
`getSteps(scenario)` can be moved to a variable, placed before `if` and then re-used twice